build.psm1

```diff
@@ -615,10 +663,40 @@ function Start-PSPester {
     else {
         try {
             $originalModulePath = $env:PSMODULEPATH
-            
-            & $powershell -noprofile -c $Command
+            if ($Unelevate)
+            {
+                Start-UnelevatedProcess -process $powershell -arguments @('-noprofile', '-c', $Command)
```

Comment:
>art-UnelevatedProcess [](start = 18, length = 21)

it's too bad that runas doesn't give you _any_ useful info about the running process